.gitignore

```diff
@@ -6,6 +6,7 @@
 node_modules/
 keycloak-*
 !keycloak-*.json
+!keycloak-connect-rest-mixed-client-spec.js
```

Comment:
I only unignored this test class for b/c unignoring all causes format errors to show up that were hidden and don't want to bloat this PR. I will send another PR cleaning up the other tests.

Reply:
@foo4u I appreciate your changes, but I don't think we should ignore it. Leave it behind will just create a technical debt for people to fix our CI in the future. We don't need to rush into things.

Reply:
@foo4u ignore this comment please. I overlooked this change.

Reply:
No problem, I’ll fix the formatting errors in the other tests today then and push another commit.

Reply:
Lol, commented too quickly myself. Will fix tests in another PR as planned.

Reply:
@foo4u which tests exactly are broken? I couldn't find any broken tests or formatting issues in your PR.

Reply:
No issues in my PR @abstractj ... problem is that some test files are in the Git but also in `.gitignore` and are therefore not checked by semistandard but do contain formatting issues.